src/backend/gporca/libnaucrates/src/parser/CParseHandlerMDRelationExternal.cpp

```diff
@@ -193,6 +193,7 @@ CParseHandlerMDRelationExternal::EndElement(
 		//		{
 		CParseHandlerScalarOp *pphPartCnstr =
 			dynamic_cast<CParseHandlerScalarOp *>((*this)[Length() - 1]);
+		GPOS_ASSERT(NULL != pphPartCnstr);
```

Comment:
I think it's better to move this change to https://github.com/arenadata/gpdb/pull/806 so that there are fewer cases when different commits do the same thing in the same method.

Reply:
moved